docs/getting_started.rst

```diff
@@ -61,8 +61,8 @@ To use :pypi:`django-celery-results` with your project you need to follow these
 
         CELERY_RESULT_EXTENDED = True
 
-    If you want to track the execution duration of your tasks (by comparing `date_created` and `date_done` in TaskResult), enable the :setting:`track_started` setting.
-    
+    If you want to track the execution duration of your tasks (by comparing `date_started` and `date_done` in TaskResult), enable the :setting:`track_started` setting.
```

Comment:
can we think of any example, using the new feature here?

Reply:
@auvipy Added example. However, I'm not good at English, so I would like you to correct the sentence.

By the way, you previously asked whether this package should include the behavior of registering TaskResult at `states.PENDING` timing.
ref: https://github.com/celery/django-celery-results/issues/286#issuecomment-1279655622

If it is possible for me, I would like to create another PR. What do you think?

Reply:
you can certainly open another PR. but what are the issues with the open PR?

Reply:
Sorry I'm not good at explaining. There is nothing problems with this PR.
I would like to submit a new PR for the PENDING status registration.
If there is no problem with this PR, would you please merge it?

Reply:
you are most welcome to come with a new PR